pkg/source/reconciler/binding/controller.go

```diff
@@ -88,7 +87,7 @@ func NewController(
 	kfkInformer.Informer().AddEventHandler(controller.HandleAll(impl.Enqueue))
 	namespaceInformer.Informer().AddEventHandler(controller.HandleAll(impl.Enqueue))
```

Comment:
Hmm, not your change, but this is a "code smell", and without a comment, I'd venture a guess that there's a bug here.  Generally there should be exactly one type/informer that queues its own key, so having two here feels off.  Happy to discuss in slack, if it'd be helpful.